controller/Previewer.php

```diff
@@ -137,15 +156,15 @@ public function init()
         try {
             $this->validateCsrf();
 
-            $serviceCallId = $this->getRequestParameter('serviceCallId');
+            $requestParams = $this->getPsrRequest()->getQueryParams();
+            $serviceCallId = $requestParams['serviceCallId'];
 
             $response = [
-                'success' => $serviceCallId == 'previewer',
+                'success' => $serviceCallId === 'previewer',
```

Comment:
this should be cast to string because of the new ''==='

Reply:
But even if it will be int, then our check will be false, and it’s also good